build.sh

```diff
@@ -6,7 +6,7 @@ usage()
     echo "managed - optional argument to build the managed code"
     echo "native - optional argument to build the native code"
     echo "The following arguments affect native builds only:"
-    echo "BuildArch can be: x64, arm"
+    echo "BuildArch can be: x64, arm, arm64"
```

Comment:
Maybe be nice to add all four (x64, x86, arm, arm64) consistently in all places while you are on it.